R/motif_enrichment_monaLisa.R

```diff
@@ -1098,6 +1100,22 @@ calcBinnedMotifEnrR <- function(seqs,
                               verbose = verbose))
     assaySumForegroundWgtWithHits <- do.call(cbind, lapply(enrichL, function(x){x$sumForegroundWgtWithHits}))
     assaySumBackgroundWgtWithHits <- do.call(cbind, lapply(enrichL, function(x){x$sumBackgroundWgtWithHits}))
+    
+    # ... set motifs with zero fore- and background sums to NA
+    assayFgBgSum <- assaySumForegroundWgtWithHits + assaySumBackgroundWgtWithHits
+    set_NA <- assayFgBgSum == 0
+    P[set_NA] <- NA
+    padj[set_NA] <- NA
+    enrTF[set_NA] <- NA
+    log2enr[set_NA] <- NA
+    assaySumForegroundWgtWithHits[set_NA] <- NA
+    assaySumBackgroundWgtWithHits[set_NA] <- NA
```

Comment:
I wonder if these two (`assaySumForegroundWgtWithHits` and `assaySumBackgroundWgtWithHits`) should be set to `NA`, because these are available and correctly set to zero. The problem is that with zeros in these, we cannot reliably calculate `P`, `padj`, `enrTF` and `log2enr`, which is why these should be `NA`.

Reply:
I agree, those two assays should remain as 0 count.